cdap-app-fabric/src/main/java/co/cask/cdap/internal/profile/ProfileService.java

```diff
@@ -236,7 +232,7 @@ public void deleteProfile(ProfileId profileId) throws NotFoundException, Profile
    * @param namespaceId the id of the namespace
    */
   @VisibleForTesting
-  public void deleteAllProfiles(NamespaceId namespaceId) {
+  void clear(NamespaceId namespaceId) {
```

Comment:
I meant for clear to delete all profiles, not just those in a namespace. It should be used to ensure the state is clean in unit tests.